remoulade/middleware/prometheus.py

```diff
@@ -134,39 +124,40 @@ def after_worker_shutdown(self, broker, worker):
         self.logger.debug("Shutting down exposition server...")
         # Do not stop it actually
 
+    def after_declare_actor(self, broker, actor):
+        # initialize the metrics for all labels to 0
+        metrics = [
+            self.worker_busy,
+            self.total_errored_messages,
+            self.total_retried_messages,
+            self.total_rejected_messages,
+            self.message_durations
+        ]
+        for metric in metrics:
+            metric.labels(actor.queue_name, actor.options.get("prometheus_label", actor.actor_name))
```

Comment:
Ok, so labels() initializes the values, no need of set(0)?

We can remove after_worker_boot (set(0)) then.

Reply:
Yeah seems more clear like this I think